app/views/groups/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<header>
+  <div class="title">
+    <div><%= link_to user_groups_path do %>
+      <%= content_tag(:div) do %>
+      <i class="material-icons">BACK</i>
+      <% end %>
+      <% end %></div>
+    <div>
+      <h2><%=@group.name%></h2>
+    </div>
+  </div>
+  <div>
+    <button><%= link_to "Add a new transaction", new_user_group_payment_path(current_user,@group), class: "button_text" %></button>
+  </div>
+</header>
+
+<div class="group_holder">
+  <div class="group_card">
+    <div class="card_element">
+      <div class="picture_holder">
+        <img src=<%= @group.icon %> alt="group picture">
+      </div>
+      <div class="name_holder">
+        <h5 class="group_name">
+          <%=@group.name%>
+        </h5>
+        <p class="detail_description">This is a short explanation for the group expenses</p>
+      </div>
+    </div>
+    <hr>
+    <div>
+      <% @payments.each do |payment| %>
+      <div class="transaction">
+        <p>Transaction No <%=payment.id%></p>
+        <div class="transaction_detail">
+          <p><%=payment.created_at.strftime('%d %B %Y')%></p>
+          <p>$<%=payment.amount%></p>
+        </div>
+      </div>
+      <% end %>
+    </div>
+    <hr>
+    <div class="total_transaction">
+      <h5 class="total_font">TOTAL PAYMENT</h5>
+      <h5 class="total_font">$<%=number_with_delimiter(@group.total_amount_by_group(@group.id))%></h5>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] As mentioned in the project requirements at the top of the page the user could see the total amount for the category (the sum of all of the amounts of transactions in that category) but your total amount is shown at the bottom. Please kindly fix that 🙏🏾 
- [ ] Also it would be better when adding a transaction to the transaction name display as well, if the first transaction name is `pants` the name should display so the user can know this transaction was for `pants` 🙃 😁

Your design 

<img width="345" alt="Screenshot 2023-06-03 at 22 56 56" src="https://github.com/MarkoKermi/Budget_app/assets/74501523/9cdc84f5-1329-449b-b8c3-795ff7e61324">